templates/rhel8.saphana.tpl.yaml

```diff
@@ -153,42 +153,55 @@ objects:
             kubernetes.io/hostname: ${TARGET_NODE_NAME}
 parameters:
   - description: Name for the new VM
+    displayName: Name
     name: NAME
     required: true
   - description: Amount of memory
+    displayName: Memory
     name: MEMORY
     value: 24Gi
   - description: Amount of memory overhead for qemu
+    displayName: Memory overhead
     name: MEMORY_OVERHEAD
     value: 44Gi
   - description: Amount of cores
+    displayName: CPU Cores
```

Comment:
nitpick: please change `Cores` to `cores` for consistency

Reply:
@lmandavi hi, do we need to capitalize here ?